### DISCUSSION TITLE
Error running salesforce_validator lambda

### DISCUSSION THREAD

**Discussion by tuyenl** [amazon-connect/amazon-connect-salesforce-scv#71] [Q&A] [answered]
I'm receiving the below error after running the salesforce_validator lambda function.
" No module named '_cffi_backend' "
I've deployed the Salesforce Config stack before with no issues, the only difference now is the sfMode parameter changed from Sandbox to Prod. I can't figure out the cause of this error, please let me know if you've experienced something similar.

**Comment by dcarbonetti** [comment, marked as answer]
Check the issues on the repo, you’ll see someone has added a copy of an older version of the python layer. Grab that zip and load it up over the top of your python layer, then point everything ro that version

**Comment by dougjaso** [comment]
This has been fixed in the latest build. If you have not deployed anything beyond this step, then you can simply rollback the template, and redeploy. If you have already deployed other resources and just want the ZIP file for the python layer, the most recent version can be [downloaded directly](https://awsscv-supporting-code-us-west-2.s3.us-west-2.amazonaws.com/awsscv_python_common.zip). This version supports both Python 3.8 and 3.9 for future releases.
